### PR TITLE
Rename lib/gambit-content

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?

Renames Gambit Campaigns as Gambit Content in anticipation of repo name change, per  https://github.com/DoSomething/gambit-campaigns/pull/1102 

#### How should this be reviewed?

Sanity test that a broadcast, replies work as expected (correct message content is returned)

#### Any background context you want to provide?

It's kind of tempting to rename this repo to just plain ol' `gambit` (once used by https://github.com/DoSomething/gambit-content, which is why https://github.com/dosomething/gambit redirects there).

#### Checklist
- [x] Documentation added for new features/changed endpoints.
- [x] Tests added/updated for new features/bug fixes.
- [x] ENV variables added/updated for new features/bug fixes.
- [x] Tested on staging.

### CODE REVIEW

**Comment by aaronschachter** [comment]
Tested on staging, and added new config vars to production.

**Comment by rapala61** [comment]
One minor leftover "campaigns" reference here: https://github.com/DoSomething/gambit-conversations/blob/master/lib/helpers.js#L29

File: config/lib/gambit-content.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 module.exports = {
   clientOptions: {
-    baseUri: process.env.DS_GAMBIT_CAMPAIGNS_API_BASEURI,
-    apiKey: process.env.DS_GAMBIT_CAMPAIGNS_API_KEY,
+    baseUri: process.env.DS_GAMBIT_CONTENT_API_BASEURI,
+    apiKey: process.env.DS_GAMBIT_CONTENT_API_KEY,
   },
   authHeader: 'x-gambit-api-key',
   endpoints: {
```
Comment (rapala61) [inline config/lib/gambit-content.js]: Should we also remove this endpoint `campaignActivity: 'campaignActivity'` ?

**Comment by aaronschachter** [comment]
Thanks for catching those details @rapala61!  Curious what you and @DFurnes think about the idea renaming this repo to just plain ol' `gambit` instead of `gambit-conversations`.

**Pull request merged**